.gitignore

```diff
@@ -1,6 +1,6 @@
-UserInterfaceState.xcuserstate
-Toucan.xccheckout
-Toucan.xcworkspace/xcuserdata/*.xcuserdatad
```

Comment:
I made this change because the previous gitignore was not catching xcuserdata changes in `Toucan.xcodeproj`.